Packs/PrismaCloud/Integrations/PrismaCloudIAM/PrismaCloudIAM_description.md

```diff
@@ -0,0 +1,2 @@
+## Prisma Cloud IAM
+- This section explains how to configure the instance of Prisma Cloud IAM in Cortex XSOAR.
```

Comment:
Please update.

Reply:
@dantavori say to we can leave it empty
i will delete the  two lines here